README.md

```diff
@@ -22,13 +22,6 @@ files and `terraform providers` to check that it has been loaded correctly.
 
 **Note**: The targeted version of Terraform is currently **v0.11.11**.
 
-## Building from source
-
-For "vanilla"-builds just do this:
-
-1. Install and configure Go
-2. `make install`
-
```

Comment:
Moved this further down because it should be less important when having more up-to-date release binaries available.